plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -2219,6 +2220,7 @@ read_only::get_table_rows_result read_only::get_kv_table_rows_context( const rea
       const string &index_value = *p.index_value;
       p.lower_bound = p.index_value;
       unbounded = true;
+      p.reverse = false;
```

Comment:
This fixes the crash when --reverse is enabled with --index_value.

Reply:
I like your review comment explaining how it fixes a particular problem.